Integrations/integration-Programmer-python.yml

```diff
@@ -0,0 +1,64 @@
+description: Programmer Life
+detaileddescription: Programmer Life
+configuration:
+- display: Server url for programmer life
```

Comment:
add a description field, to explain the format of the server url..
is it http://www.x.com/ ?
is it http://ip:port/ ?